distribution/changelog.txt

```diff
@@ -19,10 +19,13 @@
 - Feature: [#16707] [Plugin] Implement intransient plugins.
 - Feature: [#16707] [Plugin] New API for current mode, map.change hook and toolbox menu items on title screen.
 - Feature: [#16731] [Plugin] New API for fetching and manipulating a staff member's patrol area.
+- Feature: [#16740] Allow staff patrol areas to be defined with individual tiles rather than groups of 4x4.
 - Feature: [#16800] [Plugin] Add lift hill speed properties to API.
 - Feature: [#16806] Parkobj can load sprites from RCT image archives.
 - Feature: [#16831] Allow ternary colours for small and large scenery objects.
 - Feature: [#16872] [Plugin] Add support for custom images.
+- Feature: [objects#159] Invisible supports, railings & queues are selectable.
+- Feature: [objects#164] Void surface and edge are selectable.
```

Comment:
I don't know if we have a proper syntax for objects in the changelog, do we @Gymnasiast ? How do we typically incorporate those?